SofaKernel/framework/sofa/helper/system/PluginManager.cpp

```diff
@@ -289,12 +296,14 @@ std::string PluginManager::findPlugin(const std::string& pluginName, bool ignore
             {
                 const std::string& filename = *j;
                 const std::string downcaseFilename = Utils::downcaseString(filename);
-                if (downcaseFilename == downcaseLibName)
+                if (downcaseFilename == downcaseLibName) {
                     return dir + "/" + filename;
+                    std::cout<<"dir : "<<dir + "/" + filename<<std::endl;
```

Comment:
message API (msg_error/warning/etc) should be used here

Reply:
This likely result from the merging. The std::cout statement will in fact never been called, since it is after a return statement. The line should be removed entirely.